spec/api-menu-item-spec.ts

```diff
@@ -205,6 +205,22 @@ describe('MenuItems', () => {
       const canExecute = execute(item.role as any, win, win.webContents);
       expect(canExecute).to.be.true('can execute');
     });
+
+    ifit(process.platform === 'win32')('does not execute minimize role when minimizable false', () => {
```

Comment:
Wouldn't we want to make sure this is true on all platforms?

Reply:
I created an issue to better document the specifics of the bug being fixed here. So with this new code only the Windows functionality is effected. The specific test I added is only applicable to Windows since the macOS path does not involve _executeCommand when it comes to executing menu items with assigned “roles”. I could add a macOS specific test for overall test suite completeness, but in terms of testing the added functionality, this test should cover it.